### ISSUE TITLE
iPad mini 2 unable to exploit kernel. 12.1.4 (A7) unc0ver v3.3.8

### ISSUE THREAD

**Issue by supaflybro** [pwn20wndstuff/Undecimus#1224]
### Bug description:
iPad mini 2 rebooting on the first step.
Installed unc0ver 3.3.8 via cydia impactor. After the first launch it done all the steps and reported that device going to be rebooted. After that uncover rebooting my device on the first step.

### Steps to reproduce the behavior:
1. Turn on the device
2. Wait for 3-5 minutes for complete boot
3. Open unc0ver
4. Wait for "[*] Progress: Ready for jailbreak"
5. Press Jailbreak button

### Device:
 - iOS 12.1.4 (Stable) (Build: 16D57)
 - Machine Name: iPad4,5
 - Model Name: J86AP
 - unc0ver Version: 3.3.8

### **Place an "x" between the brackets if true:**
 - [ ] this is a bug others will be able to reproduce
 - [x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [ ] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

### Logs:
```
2019-07-23 02:01:38.221 Undecimus[11190:6884126] [*] unc0ver Version: 3.3.8
2019-07-23 02:01:38.222 Undecimus[11190:6884126] [*] Machine Name: iPad4,5
2019-07-23 02:01:38.222 Undecimus[11190:6884126] [*] Model Name: J86AP
2019-07-23 02:01:38.222 Undecimus[11190:6884126] [*] Kernel Version: Darwin Kernel Version 18.2.0: Wed Dec 19 20:28:49 PST 2018; root:xnu-4903.242.2~1/RELEASE_ARM64_S5L8960X
2019-07-23 02:01:38.222 Undecimus[11190:6884126] [*] Kernel Page Size: 0x1000
2019-07-23 02:01:38.222 Undecimus[11190:6884126] [*] System Version: iOS 12.1.4 (Stable) (Build: 16D57)
2019-07-23 02:01:38.222 Undecimus[11190:6884126] [*] Bundled Resources Version: 1.0~b23
2019-07-23 02:01:38.735 Undecimus[11190:6884126] [*] Status: Jailbreak
2019-07-23 02:01:38.736 Undecimus[11190:6884126] [*] Progress: Ready to jailbreak
2019-07-23 02:01:41.859 Undecimus[11190:6884126] [*] Status: Jailbreaking
2019-07-23 02:01:41.872 Undecimus[11190:6884126] [*] Progress: Exploiting kernel...
2019-07-23 02:01:41.873 Undecimus[11190:6884136] [*] KernelUtilities.c(1494): Assertion failure
2019-07-23 02:01:41.880 Undecimus[11190:6884136] [*] kCFCoreFoundationVersionNumber: 1562.000000
2019-07-23 02:01:41.880 Undecimus[11190:6884136] [*] offsets selected for iOS 12.0 or above
```
After that it reboots

**Comment by supaflybro** [comment]
Fixed in v3.4.2 Release

**Comment by armfc6161** [comment]
unc0ver Version: 4.3.1
[*] Machine Name: iPhone8,1
[*] Model Name: N71AP
[*] Kernel Version: Darwin Kernel Version 19.5.0: Wed Apr 29 21:33:51 PDT 2020; root:xnu-6153.122.1~2/RELEASE_ARM64_S8000
[*] Kernel Page Size: 0x4000
[*] System Version: iOS 13.5 (Beta) (Build: 17F5065a)
[*] Bundled Resources Version: 1.0~b26
[*] Status: Unsupported
[*] Progress: Unsupported
